data/os/RedHat.yaml

```diff
@@ -19,3 +19,4 @@ powerdns::recursor_configdir: /etc/pdns-recursor
 powerdns::recursor_config: "%{lookup('powerdns::recursor_configdir')}/recursor.conf"
 powerdns::recursor_package_name: pdns-recursor
 powerdns::recursor_service_name: pdns-recursor
+powerdns::authoritative_group: 'pdns'
```

Comment:
Please keep newlines at the end of files so that if you add another line it won't change the `authoritative_group` line. Same goes for `hiera.yaml`.